pkg/app/init.go

```diff
@@ -0,0 +1,160 @@
+package app
+
+import (
+	"fmt"
+	"os/exec"
+	"runtime"
+	"strings"
+
+	"github.com/Masterminds/semver/v3"
+	"go.uber.org/zap"
+	"helm.sh/helm/v3/pkg/cli"
+
+	"github.com/helmfile/helmfile/pkg/helmexec"
+)
+
+const (
+	HelmInstallCommand      = "curl -fsSL https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | bash"
+	HelmRequiredVersion     = "v2.10.0"
+	HelmDiffRequiredVersion = "v3.4.0"
+)
+
+type helmRequiredPlugin struct {
+	name    string
+	version string
+	repo    string
+}
+
+type HelmfileInit struct {
+	helmBinary string
+	logger     *zap.SugaredLogger
+	runner     helmexec.Runner
+}
+
+func NewHelmfileInit(helmBinary string, logger *zap.SugaredLogger, runner helmexec.Runner) *HelmfileInit {
+	return &HelmfileInit{helmBinary: helmBinary, logger: logger, runner: runner}
+}
+func (h *HelmfileInit) UpdateHelm() error {
+	return h.InstallHelm()
+}
+
+func (h *HelmfileInit) installHelmOnWindows() error {
+	windowPackageManagers := make(map[string]string)
+	windowPackageManagers["scoop"] = "scoop install helm"
+	windowPackageManagers["choco"] = "choco install kubernetes-helm"
```

Comment:
Related to https://github.com/helmfile/helmfile/pull/389/files#r990754084 - Does scoop and/or choco allows you to specify a version or version range/constraint to control which version of Helm to install?

Reply:
Idk about scoop but choco allows specify one or more versions to be installed: https://docs.chocolatey.org/en-us/choco/commands/install#examples

Reply:
Scoop can also install specific version of an app: https://github.com/ScoopInstaller/Scoop/wiki/FAQ#how-do-i-install-a-specific-version-of-an-app

Reply:
@mumoshu @lemeurherve  Thanks for your suggestions, I've dealt with it.